config/application.rb

```diff
@@ -26,9 +26,12 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
-    # Turn on the classic autoloader, because zeitwerk blows
-    # up. SRCH-2185 is the ticket to move asis to zeitwerk.
-    config.autoloader = :classic
+    config.eager_load_paths += Dir[config.root.join('lib').to_s + '/']
+    config.eager_load_paths += Dir[config.root.join('lib', 'tasks').to_s + '/']
+    config.eager_load_paths += Dir[config.root.join('lib', 'assets').to_s + '/']
+    config.eager_load_paths += Dir[config.root.join('lib', 'capistrano').to_s + '/']
+    config.eager_load_paths += Dir[config.root.join('lib', 'capistrano', 'tasks').to_s + '/']
+    config.eager_load_paths += Dir[config.root.join('app', 'workers').to_s + '/']
```

Comment:
Per our discussion, these lines don't appear to be necessary. I *believe* we can nix these as well as the following (as everything within `app/` should be autoloaded by default):
```ruby
config.autoload_paths += Dir[config.root.join('app', 'workers', '**/').to_s]
```

Reply:
Removed and re-tested locally; you are correct.